Felienne/hedy
# [BUG FIX] Sets public value to 'false' when saving a non-shared program

**Description**
Currently we set the `public` value too `None` when saving a program and it's not shared. However, as this is also an index key in Dynamo to enable us easy filtering for the `/explore` page an error is returned on the server. In this PR we fix this by always settings the value to `False` unless it is supposed to be `True`.

**Fixes**
This fix has no related issue.

**How to test**
It is not possible to test this locally as the error only occurs on the server. So deploy to alpha and verify that saving a program is working correct again.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
